std_msgs/Header header
uint16 count
uint16 rs485_error
uint16 sensor_error